voorhoede/head-start
# refactor: remove home/page route workaround

# Changes

Removes home/page route workaround in `[locale]/[...path]` as it's is no longer needed thanks to [patch in Astro v4.2.4](https://github.com/withastro/astro/blob/main/packages/astro/CHANGELOG.md#424). Thanks again @Fryuni! 🙌 

# Associated issue

N/A

# How to test

1. Open [preview link](https://refactor-locale-rest-path-ro.head-start.pages.dev/)
2. Verify home pages and sub pages both render
3. Run dev server in preview mode
4. Verify home pages and sub pages both render

# Checklist

- [x] I have made updated relevant documentation files (in project README, docs/, etc) -> removed comments
- ~~I have added a decision log entry if the change affects the architecture or changes a significant technology~~
- ~~I have notified a reviewer~~ (small patch)
